packages/transform/src/template-to-typescript.ts

```diff
@@ -148,7 +148,10 @@ export function templateToTypescript(
       }
     }
 
-    function emitComment(node: AST.MustacheCommentStatement | AST.CommentStatement): void {
+    function emitComment(
+      node: AST.MustacheCommentStatement | AST.CommentStatement,
+      templateLength?: number
```

Comment:
I think `template` itself is already in scope here?

Reply:
Indeed! 😆